examples/relative_free_energy.py

```diff
@@ -118,6 +118,7 @@ def read_from_args():
         connected_core=True,
         max_cores=1e6,
         enforce_core_core=True,
+        ring_matches_ring=True,
```

Comment:
Not for this PR: Would be nice to just have default keyword values rather than having to change this in all these different files.

Reply:
Yes, agreed. Was tempted to do this, but agree that it seems like a task for another PR.

Reply:
(Preference for explicitness was cited in previous similar suggestions for this function https://github.com/proteneer/timemachine/pull/919#discussion_r1037477106 )